public/js/lib/devtools-sham/shared/event-emitter.js

```diff
@@ -117,15 +117,15 @@ EventEmitter.prototype = {
       // event handler we're going to fire wasn't removed.
       if (originalListeners === this._eventEmitterListeners.get(aEvent) ||
           this._eventEmitterListeners.get(aEvent).some(l => l === listener)) {
-        try {
+        // try {
           listener.apply(null, arguments);
-        }
-        catch (ex) {
-          // Prevent a bad listener from interfering with the others.
-          let msg = ex + ": " + ex.stack;
-          //console.error(msg);
-          console.log(msg);
-        }
+        // }
+        // catch (ex) {
+        //   // Prevent a bad listener from interfering with the others.
+        //   let msg = ex + ": " + ex.stack;
+        //   //console.error(msg);
+        //   console.log(msg);
+        // }
```

Comment:
Was this intentional? If so, maybe delete it. 

Also, what are the places where we are still using event listeners? My understanding was that this was something we do in the old debugger as a crutch because not everything was converted to Redux. Can we phase this out?